src/FeedIo/Adapter/Guzzle/Client.php

```diff
@@ -43,7 +43,13 @@ public function __construct(\GuzzleHttp\ClientInterface $guzzleClient)
     public function getResponse($url, \DateTime $modifiedSince)
     {
         try {
-            return new Response($this->guzzleClient->request('get', $url));
+            $options = [
```

Comment:
This client is only one of two. Shouldn't `Filesystem/Client` now use `$modifiedSince` and read the file only if `mtime >= $modifiedSince`?